services/shhext/service_test.go

```diff
@@ -252,6 +253,7 @@ func (s *ShhExtSuite) TestRequestMessagesSuccess() {
 		PFSEnabled:     false,
 	}
 	service := New(shh, mock, nil, config)
+	s.NoError(service.Start(aNode.Server()))
```

Comment:
you are right, i didn't use require because it is an old test and assert was used everywhere. even where require was necessary. so i decided to be lazy)